integration/simple-long/google/protobuf/wrappers.ts

```diff
@@ -218,7 +218,7 @@ export const Int64Value = {
 
   fromJSON(object: any): Int64Value {
     const message = { ...baseInt64Value } as Int64Value;
-    message.value = object.value !== undefined && object.value !== null ? Long.fromString(object.value) : Long.ZERO;
+    message.value = object.value !== undefined && object.value !== null ? Long.fromValue(object.value) : Long.ZERO;
```

Comment:
Hm, I'm kind of surprised this line changed, b/c the only line you changed in `main.ts` was:

```
code`Long.fromValue(${from} as string)`;
```

And there is no `as string` here. Is there another change to `main.ts` that is not in the PR?

Reply:
After looking at it again, the corresponding place in `main.ts` should be line 1013. It might be a good idea to also change that to `fromValue`. But I wasn't sure if that's actually the right place. Maybe you could have a look, too.